cpp/log/cert.cc

```diff
@@ -38,6 +38,35 @@ static int X509_get_signature_nid(const X509* x) {
 
 namespace cert_trans {
 
+// Useful because util::Status clashes with Cert::Status so we have to
+// refer to it within the namespace while Cert::Status exists.
+static const util::Status& ERROR_STATUS = util::Status::UNKNOWN;
+
+// TODO. These helpers can be removed when Cert::Status has been removed.
+// They make it easier to do an incremental removal
+
+static Cert::Status StatusOrBoolToCertStatus(StatusOr<bool> status) {
```

Comment:
It's not that bad (just more work for you!), but the existence of this helper makes me notice that you've refactored "from the middle", instead of from the top or the bottom...

It's usually easier to start converting from a leaf function/method (say, `Cert::ExtensionIndex`), and work your way up to its callers, and their callers, etc. Or you can start from the top (as I went in my PRs, since I assumed you were at the bottom, figured we'd meet in the middle!), especially when there are few call sites.